migration_pipeline.py

```diff
@@ -46,6 +49,24 @@ def build_pipeline(args) -> Pipeline:
     # TODO: Add verification job at the end
 
 
+def _add_task_or_skip(pipeline, skip_tasks, task, dependent_tasks):
```

Comment:
I think it's better to add the skipping feature into pipeline framework. Similar to the dry_run. In that way, we can skipping tasks without changing pipeline structure.

Reply:
Great idea. let me do that

Reply:
done